py-polars/polars/frame.py

```diff
@@ -41,6 +43,15 @@
     from .lazy import LazyFrame, Expr
 
 
+def to_csv(df, path_or_buf=None):
```

Comment:
Could you put this logic in `DataFrame.to_csv`? I don't want `to_csv` as a different function as that would not be in line with the rest of the API.